examples/onnx/pytorch/translation/README.md

```diff
@@ -13,14 +13,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 -->
 
-# Usage
+# Bart + Beam Search to ONNX
 
-This folder contains an example of exporting Bart + Beam Search generation(BartForConditionalGeneration) to ONNX file.
+
+## Usage
```

Comment:
I'd delete this header entirely (but keep the two paragraphs below it). Usage instructions are actually in the next section.

```suggestion
```